code/game/objects/items/toys.dm

```diff
@@ -628,25 +619,17 @@
 		var/mob/living/carbon/human/H = user
 		H.set_species(/datum/species/skeleton)
 
-	phomeme = "sans"
-	toy_talk(user, "You feel like you're going to have a bad time.")
+	toy_talk(user, "RATTLE ME BONES")
 
 	user.Stun(5)
 	sleep(20)
 	return OXYLOSS
 
 /obj/item/toy/talking/skeleton/generate_messages()
-	return list(pick(papyrus_messages + sans_messages + regular_messages))
+	return list(regular_messages)
```

Comment:
TESTING IS FOR OTHER PEOPLE, GOOF IS A BUSY MAN CONTRIBUTING HIGH QUALITY CHANGES